examples/cube.go

```diff
@@ -1,13 +1,17 @@
 package main
 
 import (
+	"log"
 	"math/rand"
 
 	"github.com/fogleman/pt/pt"
 )
 
 func createMesh(material pt.Material) pt.Shape {
-	mesh, _ := pt.LoadBinarySTL("examples/cube.stl", material)
+	mesh, err := pt.LoadBinarySTL("examples/cube.stl", material)
+	if err != nil {
+		log.Fatalln(err.Error())
```

Comment:
Style suggestion. Specifying `.Error()` is not necessary since `log.Fatalln` takes care of that. It's more common to see the shorter:

```Go
if err != nil {
	log.Fatalln(err)
}
```

Or something like `log.Fatalln("some additional description:", err)` if you want to add context.